table_reconstruction/line_segmentation/line_segment.py

```diff
@@ -0,0 +1,147 @@
+import os
+
+from typing import Tuple
+
+import gdown
+import torch
+import numpy as np
+import cv2
+
+from torchvision import transforms
+from PIL import Image
+
+from .utils import load_model_unet
+
+
+MODEL_PATH = os.path.dirname(os.path.abspath(__file__)) + "/tmp/resnet_weight.pth"
+WEIGHT_URL = "https://drive.google.com/u/0/uc?id=18YEiAzUs9NXz0FwBuU0JicEWc_F2V7tq"
+
+
+class LineDetector:
```

Comment:
em nghĩ là để tên như thế này sẽ đúng chức năng hơn
```suggestion
class LineSegmentation:
```

Reply:
ok a đã sửa lại rồi nhé